regression/goto-instrument/nondet-volatile-02/test.c

```diff
@@ -0,0 +1,25 @@
+#include <assert.h>
+
+volatile int a;
+volatile int b;
+
+void func(volatile int c)
+{
+  assert(c == 0);
+  assert(c != 0);
+}
+
+void main()
+{
+  volatile int d = 0;
+  
+  assert(a == 0);
```

Comment:
:question: symetric assert a != 0?

Reply:
No - because we _don't_ havoc a - I don't understand why this passes, shouldn't a be nondet

Reply:
:peach: